src/MyEventEmitter.js

```diff
@@ -1,12 +1,87 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, callBack) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+
+    this.events[eventName].push(callBack);
+  }
+
+  once(eventName, callBack) {
+    const onceWrapper = (...args) => {
+      callBack.apply(this, args);
+
+      this.off(eventName, onceWrapper);
+    };
+
+    this.on(eventName, onceWrapper);
+  }
+
+  off(eventName, callBack) {
+    const eventCallBacks = this.events[eventName];
+
+    if (!eventCallBacks) {
+      return;
+    }
+
+    if (callBack) {
```

Comment:
It's better to throw an error or return a message when removing an event listener that does not exist.